### PR TITLE
Update to stop syncing LIMS running notes through PSUL

### PR DESCRIPTION
**Pull request by aanil**

### CODE REVIEW

File: LIMS2DB/classes.py [hunk 1]
```diff
@@ -450,6 +453,8 @@ def get_project_level(self):
         if self.project.udf_dict.get("Reference genome"):
             self.obj['reference_genome'] = self.project.udf_dict.get("Reference genome")
         self.obj['details'] = self.make_normalized_dict(self.project.udf_dict)
+        #Until the Running Notes udf is removed from lims_step
```
Comment (alneberg) [inline LIMS2DB/classes.py:456]: I guess it will be present for old projects for a long time still?

Comment (aanil) [reply LIMS2DB/classes.py]: Removed comment!

**Review by pericsson — APPROVED** [review]
Looks good to me

**Review by alneberg — APPROVED** [review]
Looks good!

**Pull request merged**